### PR TITLE
updated pre-commit

### PR DESCRIPTION
**Pull request by Goldziher**

### CODE REVIEW

**Review by sondrelg — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -12,21 +12,20 @@ repos:
       - id: trailing-whitespace
       - id: check-ast
       - id: check-json
+      - id: check-yaml
       - id: check-merge-conflict
-      - id: no-commit-to-branch
```
Comment (Goldziher) [inline .pre-commit-config.yaml:16]: one doesnt have privileges or one knows what he is doing :)

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -12,21 +12,20 @@ repos:
...
-        args: [ --branch, master ]
   - repo: https://gitlab.com/pycqa/flake8
     rev: 3.8.4
     hooks:
       - id: flake8
         additional_dependencies: [
-            'flake8-bugbear==20.11.1',
```
Comment (Goldziher) [inline .pre-commit-config.yaml:23]: removed pinned - i assume the latest version works for us

Comment (sondrelg) [reply .pre-commit-config.yaml:23]: Yep agree

File: .pre-commit-config.yaml (hunk 1 above)
Comment (sondrelg) [reply .pre-commit-config.yaml:16]: Just fyi, you can always skip pre-commit by running `--no-verify` when you commit, so not sure this needs to be removed

Comment (Goldziher) [reply .pre-commit-config.yaml:16]: im aware but i do not like -n

**Pull request merged**